crates/rspack_core/src/compiler/rebuild.rs

```diff
@@ -190,6 +190,38 @@ impl Compiler {
     self.compile_done().await?;
     self.cache.after_compile(&self.compilation).await;
 
+    #[cfg(allocative)]
```

Comment:
do we need add this to build_inner?

Reply:
if we add both to build and rebuild maybe move it to init(like sftrace) is better?